test/integration/storage/test_objects.rb

```diff
@@ -0,0 +1,114 @@
+require "helpers/integration_test_helper"
+
+class TestObjects < FogIntegrationTest
+  def setup
+    # Uncomment this if you want to make real requests to GCE (you _will_ be billed!)
```

Comment:
Unnecessary comment in integration tests, ditto for the one in buckets and objects.